python/lsst/ts/scriptqueue/ui/script_queue_commander.py

```diff
@@ -31,17 +32,43 @@
 
 
 class ScriptQueueCommander(salobj.CscCommander):
-    def __init__(self, **kwargs):
+    """ScriptQueue command-line commander.
+
+    Parameters
+    ----------
+    script_log_level : `int`
+        Default log level for scripts.
+    """
+
+    def __init__(self, script_log_level, **kwargs):
         super().__init__(name="ScriptQueue", **kwargs)
+        self.script_log_level = script_log_level
         self.help_dict[
             "add"
-        ] = """type path config  # add a script to the end of the queue:
+        ] = f"""type path config options  # add a script to the end of the queue:
     • type = s or std for standard, e or ext for external
-    • config = @yaml_file_path or keyword1=value1 keyword2=value2 ..."""
+    • config = @yaml_file_path or keyword1=value1 keyword2=value2 ...
+    • options can be any of the following, in any order
+      (but all option args must follow all config args):
+      -location=int  # first=0, last=1, before=2, after=3
+      -locationSalIndex=int
+      -logLevel=int  # error=40, warning=30, info=20, debug=10; default is {script_log_level}
+      -pauseCheckpoint=str  # a regex
+      -stopCheckpoint=str  # a regex"""
```

Comment:
I think it would be helpful if you could add a couple of examples of an add command setting some values.. something like:

```
Example:

add s some_std_script.py @str_script_config_1.yaml -logLevel=10

add s some_std_script.py value1=10 value2=string_value -logLevel=10
```

Reply:
Good suggestion. I did that.